lib/kubeclient_next/client/rest_client.rb

```diff
@@ -5,18 +5,18 @@
 module KubeclientNext
   module Client
     class RESTClient
-      attr_reader :group, :version
+      attr_reader :context, :path
```

Comment:
This is part of moving `RESTClient` into being less 'smart', allowing for easy instantiation of clients targeting arbitrary paths, should the need arise